libs/atqr/domain/src/lib/domain-spec/stories/judgeAChallenge.disl

```diff
@@ -0,0 +1,14 @@
+read_also api.disl
+read_also enums.disl
+
+ 💬 judgeChallenge? :challengeId, accomplished: boolean
+
+        @challengeJudgement = ChallengeJudgement.Judged
```

Comment:
I think the challengeJudgement has to be "Requested" , not "judged"

Reply:
actually, both happen. We first request, than the judge judges :)